whois-nrtm4/src/main/java/net/ripe/db/nrtm4/generator/UpdateNotificationFileGenerator.java

```diff
@@ -168,12 +163,12 @@ private List<UpdateNotificationFile.NrtmFileLink> getPublishableFile(final List<
     private UpdateNotificationFile.NrtmFileLink getPublishableFile(final NrtmVersionInfo versionInfo, final String file, final String hash) {
         return new UpdateNotificationFile.NrtmFileLink(
                 versionInfo.version(),
-                urlString(versionInfo.source().getName().toString(), file),
+                urlString(file),
                 hash);
     }
 
-    private String urlString(final String source, final String fileName) {
-        return String.format("%s/%s", source, fileName);
```

Comment:
Are there any testcase where we read the snapshot and delta file from the UNF rleative path ? Test should use the same UNF url and add the relative path and fetch the files